sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/implementation/models/CancelJobResultInternal.java

```diff
@@ -5,17 +5,53 @@
 
 import com.azure.core.annotation.Generated;
 import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
 
 /**
  * Response payload from cancelling a job.
  */
 @Immutable
-public final class CancelJobResultInternal {
+public final class CancelJobResultInternal implements JsonSerializable<CancelJobResultInternal> {
```

Comment:
Just curious, seems like most of the internal classes are removed.  Is there something specific about these classes that make it not possible to remove?

Reply:
I really never figured out what to do with this class and the similar `Internal` classes as they're defined in the TypeSpec without any properties but are used in service methods that are hidden from the public API, so I left them. These can be followed up on if they should be removed.